charts/hono/config/router/qdrouterd.json

```diff
@@ -36,6 +36,7 @@
     "saslPlugin": "Hono Auth"
   }],
 
+{{- if .Values.amqpMessagingNetworkExample.insecurePortEnabled }}
```

Comment:
This condition can also be placed around the definition of the `amqp` port 5672 in `dispatch-router-deployment.yaml`.